### PR TITLE
getting bot's token

### PR DESCRIPTION
**Pull request by oMvhmoud**
The function has been more user-friendly and getting bot's token feature.

### CODE REVIEW

**Comment by v8j** [comment]
don't forget
every account just make 25 account if you need more you can use the authy api

File: index.js [hunk 1]
```diff
@@ -1,30 +1,41 @@
 const { default: axios } = require("axios");
-const config = require('./config.json')
+const config = require("./config.json");
```
Comment (FnrDev) [inline index.js]:
```suggestion
/**
 * @param {string} token - The account token 
 * @param {string} name - The bot name to create
 */
```

File: index.js [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
-    const botData = createBot.data;
-    console.log(botData)
+async function generateBot(token, name) {
+  const req = await axios({
+    url: "https://discord.com/api/applications",
+    method: "POST",
+    headers: {
+      Authorization: token,
+    },
+    "content-type": "application/json",
```
Comment (FnrDev) [inline index.js]:
```suggestion
```

File: index.js [hunk 3]
```diff
@@ -1,30 +1,41 @@
...
+  const data = req.data;
+  console.log(data);
+    
+  await axios({
+    url: `https://discord.com/api/applications/${data.id}/bot`,
+    method: "POST",
+    headers: {
+      Authorization: token,
+    },
+    "content-type": "applicaction/json",
```
Comment (FnrDev) [inline index.js]:
```suggestion
```

File: index.js [hunk 4]
```diff
@@ -1,30 +1,41 @@
...
+  }).catch(console.error);
+
+  const getToken = await axios({
+    url: `https://discord.com/api/applications/${data.id}/bot/reset`,
+    method: "POST",
+    headers: {
+      Authorization: token,
+    },
+    "content-type": "applicaction/json",
```
Comment (FnrDev) [inline index.js]:
```suggestion
```

File: index.js [hunk 5]
```diff
@@ -1,30 +1,41 @@
...
+  }).catch(console.error);
+
+  console.log(getToken.data);
 }
 
-getData()
\ No newline at end of file
+generateBot(config.token, "botname"); // first parameter (account's token), second parameter (bot's name)
```
Comment (FnrDev) [inline index.js]:
```suggestion
generateBot(config.token, "someCoolName");
```

**Review by FnrDev — CHANGES_REQUESTED** [review]
👋 hey there, I have some requested changes please review them.

Thanks for contributing

**Review by FnrDev — APPROVED** [review]

**Comment by FnrDev** [comment]
> don't forget every account just make 25 account if you need more you can use the authy api

Yeah, I agree we should do something to handle the current bot size in the account.

**Comment by v8j** [comment]
you can use authy api to make the bots on team's

**Comment by FnrDev** [comment]
> you can use authy api to make the bots on team's

We don't need that, we can just make a new request to make a discord team

**Pull request merged**